cmd/deploy/start_test.go

```diff
@@ -98,6 +98,72 @@ func (suite *DeployStartTestSuite) TestDeployStartYAMLSuccess() {
 	suite.Equal(expected.PipelineID, received.DeploymentId, "they should be equal")
 }
 
+func (suite *DeployStartTestSuite) TestDeployStartWithURLSuccess() {
+	expected := &de.StartPipelineResponse{
+		PipelineID: "12345",
+	}
+	err := registerResponder(expected, http.StatusAccepted)
+	if err != nil {
+		suite.T().Fatalf("TestDeployStartWithURLSuccess failed with: %s", err)
+	}
+	outWriter := bytes.NewBufferString("")
+	cmd := getDeployCmdWithFileName(outWriter, "https://myhostedfile.example.com/deploy.yaml", "yaml")
+	err = cmd.Execute()
+	if err != nil {
+		suite.T().Fatalf("TestDeployStartYAMLSuccess failed with: %s", err)
+	}
+	output, err := ioutil.ReadAll(outWriter)
+	if err != nil {
+		suite.T().Fatalf("TestDeployStartYAMLSuccess failed with: %s", err)
+	}
+	var received = FormattableDeployStartResponse{}
```

Comment:
```suggestion
	var received FormattableDeployStartResponse
```

^^^ same thing and more idiomatic